content/analyze/portals/asap.md

```diff
@@ -0,0 +1,9 @@
+---
+path: "/analyze/portals/asap"
+date: "2018-05-03"
+title: "Automated Single cell Analysis Platform (ASAP) | Vincent Gardeux, Fabrice David, Bart Deplancke"
+subTitle: "SAP allows the user to perform custom analyses and compare algorithms for each step of the single cell or bulk RNA-seq analysis pipeline post genome alignment via an intuitive web interface (Gardeux et al., Bioinformatics, 2017). These steps include parsing, filtering, and normalization of the input gene expression matrix, visual (2D and 3D) representation, differential expression, clustering, heatmaps, trajectory inference and functional enrichment analyses to characterize novel cell clusters, specific cell types, or differentiation processes. Thus, ASAP has been developed to lower the bioinformatic entry level to single cell experiments and to catalyze collaborations between computational biologists and experimentalists via an easy-to-use data interaction portal."
+appUrl: "https://asap.epfl.ch/"
+githubUrl:: "https://asap.epfl.ch/"
```

Comment:
@NoopDog, I've kept in `githubUrl` for portal apps for the time being, but this field is probably not essential for list items in the Portals tab.  The new `appUrl` field could bring the user to the listed portal app.

Reply:
Expanding on that: our portals are best construed as "apps", whereas methods and visualization components are "packages" (that can be used in apps).  Portals are targeted at researchers and end users; methods and visualization components are targeted more at engineers.